localstorage.js

```diff
@@ -0,0 +1,24 @@
+const userName = document.getElementById('name');
+const email = document.getElementById('email');
+const userMessage = document.getElementById('comments');
+const form = document.getElementById('contact_form');
+
+const formInputsData = {
+  name: '',
+  email: '',
+  message: '',
+};
+form.addEventListener('input', () => {
+  formInputsData.name = userName.value;
+  formInputsData.email = email.value;
+  formInputsData.message = userMessage.value;
+  localStorage.setItem('formInputsData', JSON.stringify(formInputsData));
+});
+window.onload = () => {
+  const data = JSON.parse(localStorage.getItem('formInputData'));
+  if (data) {
+    userName.value = formInputsData.name;
+    email.value = formInputsData.email;
+    userMessage.value = formInputsData.message;
+  }
```

Comment:
- [ ] Nice adding the logic to preserve data on the browser but when I reload the page I don't  see what typed so please make sure to fix that so that when you reload the page you should see the info that you typed before